sbt-reladomo-plugin/src/test/resources/src_unmanaged/kata/domain/scala_api/AllTypes.scala

```diff
@@ -55,10 +55,10 @@ case class NewAllTypes(id: Int, booleanValue: Boolean, byteValue: Byte, shortVal
       case Some(_nullableDoubleValue) => underlyingObj.setNullableDoubleValue(_nullableDoubleValue)
       case _ => underlyingObj.setNullableDoubleValueNull()
     }
-    underlyingObj.setNullableDateValue(nullableDateValue.orNull[java.util.Date])
-    underlyingObj.setNullableTimestampValue(nullableTimestampValue.orNull[java.sql.Timestamp])
-    underlyingObj.setNullableStringValue(nullableStringValue.orNull[String])
-    underlyingObj.setNullableByteArrayValue(nullableByteArrayValue.orNull[Array[Byte]])
+    underlyingObj.setNullableDateValue(nullableDateValue.map(_nullableDateValue => _nullableDateValue).orNull[Date])
+    underlyingObj.setNullableTimestampValue(nullableTimestampValue.map(_nullableTimestampValue => _nullableTimestampValue).orNull[Timestamp])
+    underlyingObj.setNullableStringValue(nullableStringValue.map(_nullableStringValue => _nullableStringValue).orNull[String])
+    underlyingObj.setNullableByteArrayValue(nullableByteArrayValue.map(_nullableByteArrayValue => _nullableByteArrayValue).orNull[byte[]])
```

Comment:
```scala
scala> None.orNull[byte[]]
<console>:1: error: identifier expected but ']' found.
None.orNull[byte[]]
                 ^
```

`byte[]`  must be `Array[Byte]` in Scala.

Reply:
Thanks!
https://github.com/laysakura/reladomo-scala/pull/1/files#diff-b7d7f2dc639b2ac1018db156121ee8c7R519